### DISCUSSION TITLE
Different layouts of link orderings

### DISCUSSION THREAD

**Discussion by nobaraos12** [JulianPrieber/littlelink-custom#244] [Feature requests]
Having different layouts of link orderings would be cool, instead of a simple list.

Some ideas 

- Squares side by side 4x4
Example: 
![Screenshot from 2022-11-30 01-12-48](https://user-images.githubusercontent.com/118019611/204721358-e8dfdb19-526b-49bb-b0a8-1aaf026d4c90.png)

- Triangles side by side, stacked 
Example:
![Screenshot from 2022-11-30 01-14-54](https://user-images.githubusercontent.com/118019611/204721716-439e560d-988e-4feb-8efb-989fa6788c61.png)

- Circles

**Comment by JulianPrieber** [comment]
### [**Themes**](https://littlelink-custom.com/themes.php)

**Reply by JulianPrieber** [reply]
But seriously, I have the intention to use a theme to do something similar at some point, however my frontend coding skills are rather lacking.

**Reply by JulianPrieber** [reply]
But just to make this clear, this is nothing I would consider for the default theme and or app behavior.

**Reply by nobaraos12** [reply]
well for "rather lacking" littlelink custom got alot of great features
